File: misc/ruby/fizzbuzz.rb
input = ARGV[0]

def div_by(number, divisor)
  if number.to_i % divisor == 0
    return true
  end
end

def check_fizzinass(number)
  puts "fizzbuzz" if div_by(number, 15)
  puts "fizz" if div_by(number, 3)
  puts "buzz" if div_by(number, 5)
  puts "#{number} :("
end

check_fizzinass(input)

File: misc/js/fizzbuzz.js
function divBy(number, divisor) {
  if (number % divisor == 0) {
    return true
  }
}

function checkFizzinass(number) {
   if (divBy(number, 15)) {
     console.log("fizzbuzz");
   } else if (divBy(number, 5)) {
     console.log("buzz");
   } else if (divBy(number, 3)) {
     console.log("fizz");
   } else {
    console.log(":(", number);
   }
}

input = process.argv[2];

checkFizzinass(input);
